providers/burst-sms/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/burstSms.provider';
```

Comment:
```suggestion
export * from './lib/burst-sms.provider';
```
I missed this in my previous review. Apologies for that.